pygeoapi/provider/postgresql.py

```diff
@@ -200,16 +200,23 @@ def query(self, startindex=0, limit=10, resulttype='results',
         with DatabaseConnection(self.conn_dic, self.table) as db:
             cursor = db.conn.cursor(cursor_factory=RealDictCursor)
             sql_query = SQL("DECLARE \"geo_cursor\" CURSOR FOR \
-             SELECT {0},ST_AsGeoJSON({1}) FROM {2}").\
```

Comment:
Aside: is it possible to use pylint in this project, and enforce such rules with pylint? There's a couple of idioms where there's a matter of preference and it would save effort and discussion in the longer-term to use a linter.

Reply:
> There is only one small change missing, for approval, the formating as {} and not as {0}, think we are only implementing {}

Will do.

Reply:
Done.